doc/tutorial/index.rst

```diff
@@ -0,0 +1,198 @@
+.. _tutorial:
+
+===============
+Sphinx tutorial
+===============
+
+In this tutorial you will build a simple documentation project using Sphinx,
+and view it in your browser as HTML.
+The project will include narrative, handwritten documentation,
+as well as autogenerated API documentation.
+
+The tutorial is aimed towards Sphinx newcomers
+willing to learn the fundamentals of how projects are created and structured.
+You will create a fictional software library to generate random food recipes
+that will serve as a guide throughout the process,
+with the objective of properly documenting it.
+
+To showcase Sphinx automatic documentation generation capabilities
+you will use Python, which is the default :term:`domain`
+(even though several other languages are supported,
+they all work in a similar way).
+
+To follow the instructions you will need access to a Linux-like command line
+and a basic understanding of how it works,
+as well as a working Python installation for development,
+since you will use *Python virtual environments* to create the project
+(you can read more about them in the `Python Packaging User Guide`_).
+
+.. _Python Packaging User Guide: https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment
+
+Getting started
+---------------
+
+Setting up our project and development environment
+..................................................
```

Comment:
We usually uses `~~~~` for the 3rd level headings. Could you use it?

Reply:
Sure, didn't notice that. Changing!